docs/hardware/devices/index.mdx

```diff
@@ -25,6 +25,13 @@ Please do your research and choose the board that meets your needs (or maybe alr
 - ESP32-based devices require more power to operate but are typically lower-cost alternatives that do perform well when using house power, or for handsets that only require a day or two of runtime, and for applications that require WiFi connectivity.
 :::
 
+### [CanaryOne](./canary/)
+Complete solution with battery, screen, case, and antenna.  Ships pre-flashed with latest Meshtastic firmware.
+
+| Name                                           | MCU      | Radio  |     WiFi     | BT  |  GPS   |
+|:-----------------------------------------------|:---------|:-------|:------------:|:---:|:------:|
+| [CanaryOne](./canary/canary-one)               | nRF52840 | SX1262 |      NO      | 5.0 | YES    |
```

Comment:
I think is is a broken link

Reply:
@pdxlocations thanks for the quick review.  fixed.